examples/upload-files/README.MD

```diff
@@ -0,0 +1,25 @@
+# Select Files To Upload
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+TestCafe cannot access the browser's `Choose File` dialog to emulate file selection. For this reason, TestCafe uses another way to select files. This example shows how to select files to populate a `type="file"` input.
+
+The sample page includes a `<form>` that contains:
+
+* an `<input>` with `type="file"` that holds the selected files
+* an `<input>` with `type="submit"` that sends the files to the server
+
+During the test the `t.setFilesToUpload` method sets the files for the `<input type="file">`. A click on the `<input type="submit">` initiates the file upload to the server. The server responds with a list of uploaded files, which is checked with the `t.expect.eql` Method.
```

Comment:
> A click on the `<input type="submit">` initiates the file upload to the server.

It's not really bad, but still, a little (quite a bit) weird to say that a *click does something* in technical docs. It's kind of figurative a bit.

*The test clicks the `<input type="submit">` to initiate the file upload.*

Reply:
Edited